bob/aggregate.go

```diff
@@ -116,6 +116,9 @@ func (b *B) Aggregate() (aggregate *bobfile.Bobfile, err error) {
 		return nil, usererror.Wrap(ErrCouldNotFindTopLevelBobfile)
 	}
 
+	decorations, err := collectDecorations(aggregate)
```

Comment:
The reason I didn't implemented it there was the comment:

```go
// AggregateSparse reads Bobfile with the intent to gather task names.
// The returned bobfile is not ready to be executed with a playbook.
```

Is this still up to date? Because if AggregateSparse is only for collecting task names we should not be interested in decorations

Reply:
Ok, agree.